en/core-libraries/events.rst

```diff
@@ -466,6 +466,26 @@ params you used for attaching it::
     // Detaching all callbacks implemented by a listener
     $this->getEventManager()->detach($listener);
 
+Core events
+===========
+
+There are a number of core events within the framework which your application can listen to.
+
+* beforeMarhsal
+* beforeFind
+* buildValidator
+* buildRules
+* beforeRules
+* afterRules
+* beforeSave
+* afterSave
+* afterSaveCommit
+* beforeDelete
+* afterDelete
+* afterDeleteCommit
```

Comment:
These should have the `Model.` prefix on them. Right now these event names are never fired.